array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map((number) => number * 2);
+console.log(doubled);
+
+const prices = numbers.map((number) => `$${number.toFixed(2)}`);
+console.log(prices);
+
+const upperCased = languages.map((language) => language.toUpperCase());
+console.log(upperCased);
+
+const firstLetters = languages.map((language) => language.charAt(0));
```

Comment:
MDN now says that `language[0]` works the same as `language.charAt(0)` and is the "more modern" syntax. So you probably should change this to `language[0]`.